projects/components/src/description/description.component.ts

```diff
@@ -71,8 +71,11 @@ export class DescriptionComponent implements OnChanges, AfterViewInit {
   }
 
   public remeasure(): void {
-    this.isDescriptionTruncated$.next(
-      this.eventDescriptionContainer.nativeElement.offsetWidth < this.fullDescriptionTextWidth
-    );
+    this.isDescriptionTextToggled = false;
+    this.cdRef.detectChanges();
+
+    this.isDescriptionTruncated =
+      this.eventDescriptionContainer.nativeElement.offsetWidth < this.eventDescriptionText.nativeElement.scrollWidth;
+    this.cdRef.detectChanges();
```

Comment:
Only run once, at the end of the remeasure method.

Reply:
It will cause bug, value of `isDescriptionTruncated` will be wrong as we try to get `this.eventDescriptionText.nativeElement.scrollWidth` before it truncated and DOM changed.

Reply:
Hmm - that's one of the weird parts about using `detectChanges` explicitly instead of `markForCheck` (which just invalidates this node in the change detection tree to be picked up by angular's general change detection, which should ensure that the changes settle. Changing that here may be better - could you try that out, and if it's not working we'll go with this?

Reply:
I try to use `markForCheck` method but unfortunately it not work correctly